core/src/main/scala/chisel3/experimental/verification/package.scala

```diff
@@ -13,19 +13,21 @@ package object verification {
     def apply(predicate: Bool, msg: String = "")(
       implicit sourceInfo: SourceInfo,
       compileOptions: CompileOptions): Unit = {
-      val clock = Builder.forcedClock
-      pushCommand(Verification(Formal.Assert, sourceInfo, clock.ref,
-        predicate.ref, msg))
+      when (!Module.reset.asBool) {
+        val clock = Builder.forcedClock
```

Comment:
```suggestion
      when (!Module.reset.asBool) {
        val clock = Module.clock
```
`Module.clock` is `Builder.forcedClock` (and `Module.reset` is `Builder.forcedReset`), so ought to be consistent here.